schematools/cli.py

```diff
@@ -466,31 +466,58 @@ def create_identifier_index(schema_url, db_url):
 @create.command("tables")
 @option_db_url
 @option_schema_url
-def create_tables(schema_url, db_url):
-    """Execute SQLalchemy Table objects"""
+@click.option(
+    "--relname-from-identifier/--no-relname-from-identifier",
+    default=False,
+    help="Derive relation column name from `identifier` property in schema.",
+)
+def create_tables(schema_url: str, db_url: str, relname_from_identifier: bool):
+    """Execute SQLalchemy Table objects.
+
+    Without --relname-from-identifier all column names used to capture relations will have
```

Comment:
Goed plan om dit toe te voegen! Dan maken we al een migratiepad naar de uiteindelijke gewenste situatie.